### ISSUE TITLE
Add NIfTI file format support

### ISSUE THREAD

**Issue by a-y-khan** [SCIInstitute/SCIRun#655]
Write importer and exporter for the [NIfTI file format](http://nifti.nimh.nih.gov/nifti-1/documentation/faq). NIfTI library with CMake file is [available](http://afni.nimh.nih.gov/pub/dist/src/nifti/).

**Comment by pip010** [comment]
How is this going?
I can help you this one! I need to do it for my scirun4 clone anyway!

**Comment by aylakhan** [comment]
Not started yet. Seg3D supports NIfTI, or you can use ITK, or other Kitware tools.

**Comment by pip010** [comment]
Indeed, I looked into ITK nifti reader quite recently and looked quite
capable. However, cant say the same for VTK.
Can also look into Seg3D with my sample repo of nifti files.
Thanks.

On Tue, Aug 25, 2015 at 5:59 PM, Ayla Khan <notifications@github.com> wrote:

> Not started yet. Seg3D supports NIfTI, or you can use ITK, or other
> Kitware tools.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SCIInstitute/SCIRun/issues/655#issuecomment-134648055>
> .
>



-- 
All the best,
Petar Petrov
http://ppetrov.net

**Comment by dcwhite** [comment]
@jessdtate Any other user examples for this one?

**Comment by jessdtate** [comment]
Butson's students use this format.  We can ask them for some examples.

**Comment by dcwhite** [comment]
@SCIInstitute/butson-lab Anybody want this feature implemented for current work?

**Comment by ChantelC** [comment]
[DBS1_postMR.nii.gz](https://github.com/SCIInstitute/SCIRun/files/3790145/DBS1_postMR.nii.gz)

**Comment by dcwhite** [comment]
**Triage 2026-06:** NIfTI file format support — a NIfTI importer/exporter would be useful for neuroimaging workflows. The NIfTI library has CMake support. No progress since filing. Medium effort; would require a new Field I/O plugin similar to the existing DICOM path. Higher priority if neuroimaging users are active.
